app/controllers/thredded/private_topics_controller.rb

```diff
@@ -66,6 +66,12 @@ def update
       end
     end
 
+    def mark_all_read
+      MarkAllRead.run(Thredded::PrivateTopic, thredded_current_user) if signed_in?
```

Comment:
I don't think we'd need to inject the `Thredded::PrivateTopic` class here because there aren't plans to run this against regular topics (I don't think?). What do you think? Worth keeping around for normal topics? Since we don't keep a long running count shown to users for regular topics it's not as desirable as having it for the private topics.